SourceKitten.xcworkspace/xcshareddata/SourceKitten.xcscmblueprint

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
I think you're both right, that this shouldn't be checked in. My current theory is that Xcode puts this in the shared directory in case you get a project without `.git/`, so you can retrace the dependencies.